astro-i18next.config.mjs

```diff
@@ -2,14 +2,4 @@
 export default {
   defaultLocale: "de",
   locales: ["de", "en"],
-  load: ["client", "server"],
-
-  /** Workaround for for leading '/' in path. Will be fixed in next release.
-   * @see https://github.com/yassinedoghri/astro-i18next/issues/79
-   */
-  i18nextServer: {
-    backend: {
-      loadPath: "./public/locales/{{lng}}/{{ns}}.json",
-    },
-  },

```

Comment:
The bug below seems to be fixed indeed. I still need the `load: ["client", "server"]` however, to access the translations in react :)